packages/manager-components/src/components/ManagerButton/ManagerButton.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { PropsWithChildren } from 'react';
+import {
+  OsdsButton,
+  OsdsTooltip,
+  OsdsTooltipContent,
+} from '@ovhcloud/ods-components/react';
+import { useTranslation } from 'react-i18next';
+import './translations';
+
+import { useIsAuthorized } from '../../hooks/iam';
+
+export type ManagerButtonProps = PropsWithChildren<{
+  action?: string;
+  urn?: string;
+}>;
+
+export const ManagerButton = ({
+  children,
+  action,
+  urn,
+  ...restProps
+}: ManagerButtonProps & any) => {
+  const { t } = useTranslation('iam');
+  const isAuthorized = useIsAuthorized(action, urn);
+  if (!isAuthorized) {
+    return (
+      <OsdsTooltip>
+        <OsdsButton disabled={isAuthorized ? undefined : true} {...restProps}>
```

Comment:
**nitpick:** This must be `true` always.
```suggestion
        <OsdsButton disabled=true {...restProps}>
```

Reply:
more clean 👍

Reply:
useless I delete it